server/src/api/modules/players/player.service.js

```diff
@@ -27,22 +27,26 @@ function formatUsername(username) {
     .join(' ');
 }
 
-function isValidUsername(username) {
-  const formattedUsername = formatUsername(username);
+function isValidUsernames(members) {
```

Comment:
Try not to use the term member for this as "member" is a term used for groups, and since you're using this method for competitions, which uses "participant", it's incorrect. Maybe "usernames" is a more generic variable name.

**Also, is there a reason why you're changing this method to accept multiple values, instead of doing something like:** 
`
const invalidUsernames = usernames.filter(playerService.isValidUsername);
`

This way, the isValidUsername function could be used for single usernames in the future,

Reply:
Posting what I said in Discord for historical sake:

> The reason I changed it that way is that there’s no reason you should need to call that function X number of times for a validation when you can send all the data once, loop, and return whatever is bad back.
You can still do a single check by passing in a single name in an array if you want still.